lib/ansible/modules/source_control/gitlab_project_variable.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Markus Bergholz (markuman@gmail.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+module: gitlab_project_variable
+short_description: Creates/updates/deletes Gitlab Projects Variables
+description:
+  - When a project variable does not exists, it will be created.
+  - When a project variable does exists, its value will be updated when the values are different.
+  - Variables which are presented in the playbook, but are not presented in the Gitlab project,
+    they stay untouched (purged_vars = False) or will be deleted (purged_vars = True).
+version_added: "2.9"
+author:
+    - "Markus Bergholz (@markuman)"
+requirements:
+  - python >= 2.7
+  - python-gitlab python module
+options:
+  state:
+    description:
+      - create or delete project variable.
+      - Possible values are present and absent.
+    default: present
+    type: str
+    choices: ["present", "absent"]
+  server_url:
+    description:
+      - The URL of the Gitlab server, with protocol (i.e. http or https).
+    required: true
+    type: str
+  login_token:
+    description:
+      - Gitlab access token with api permissions.
+    required: true
+    type: str
+  name:
```

Comment:
Change this parameter for a more explicite one : `project`
This one should handle project `path` or `id`

Reply:
https://github.com/markuman/ansible/commit/c2de76da386ebbc6770acca6dfdfe98bae57db87  
https://github.com/markuman/ansible/commit/ae0157ddc21bab5120c1e71a0cb8c50c686c7c82#diff-cde424ddaa0e494488bbf72147a63aa8R203